activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/BaseBpmnXMLConverter.java

```diff
@@ -400,7 +402,12 @@ protected void writeEventDefinitions(Event parentEvent, List<EventDefinition> ev
     }
   }
 
-  protected void writeTimerDefinition(Event parentEvent, TimerEventDefinition timerDefinition, XMLStreamWriter xtw) throws Exception {
+    protected void writeLinkDefinition(Event parentEvent, LinkEventDefinition eventDefinition, XMLStreamWriter xtw) throws Exception {
+    xtw.writeStartElement(ELEMENT_EVENT_LINKDEFINITION);
+    writeDefaultAttribute(ATTRIBUTE_ID, eventDefinition.getId(), xtw);
```

Comment:
There is some missing logic here to add source and target elements for instance. Some test coverage would help with that

Reply:
Added